client/src/components/menuCategoryButtons.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
 const MenuCategoryButtons = props => (
```

Comment:
Destructure props when you invoke the component function.
```suggestion
const MenuCategoryButtons =({
  restaurants,
  currentSubMenu,
  handleMenuCategoryClick,
})=> (
```

Reply:
this props destructure suggestion is really helpful